gpflow/conditionals.py

```diff
@@ -17,13 +17,95 @@
 
 from . import settings, mean_functions
 from .decors import name_scope
-from .features import InducingPoints
+from .dispatch import conditional, sample_conditional
 from .expectations import expectation
+from .features import Kuu, Kuf, InducingPoints, InducingFeature
+from .kernels import Kernel, Combination
 from .probability_distributions import Gaussian
 
+float_type = settings.float_type
```

Comment:
This means the functions in here will only use the value from gpflowrc at startup, and not support run-time changes to float_type or jitter_level - would be better to use settings.float_type etc.